build.sbt

```diff
@@ -716,22 +717,26 @@ lazy val kernelDefaults = (project in file("kernel/kernel-defaults"))
   ).configureUnidoc(docTitle = "Delta Kernel Defaults")
 
 
-lazy val sparkKernelDsv2 = (project in file("spark-dsv2"))
+lazy val sparkKernelDsv2 = (project in file("spark-kernel-dsv2"))
   .enablePlugins(ScalafmtPlugin)
   .dependsOn(kernelApi)
   .dependsOn(kernelDefaults)
   .dependsOn(spark % "test->test")
   .settings(
     name := "delta-spark-dsv2",
     commonSettings,
+    scalaStyleSettings,
```

Comment:
Can you do `scalafmtCheckSettings`? I think just `scalaStyleSettings` doesn't enable fmt